recipes/pheasant/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pheasant" %}
+{% set version = "2.5.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0466da0a94eb2324dab43e4583b1251891adb012a6d264aaed42c7813cbfd07e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >3.7
+    - pip
+    - setuptools
+  run:
+    - python >3.7
+
+test:
+  imports:
+    - pheasant
+
+about:
+  home: https://github.com/daizutabi/pheasant
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Documentation tool for Markdown conversion by Jupyter client'
+  doc_url: https://pheasant.daizutabi.net/
+  dev_url: https://github.com/daizutabi/pheasant
+
+extra:
+  recipe-maintainers:
+    - daizutabi
```

Comment:
Please confirm that the co-maintainer agrees to be listed

Reply:
@daizutabi as the author of package pheasant, are you ok to be listed as a maintainer of the conda-forge package ?

Reply:
Yes, I'm OK.